settings.php

```diff
@@ -114,6 +114,16 @@
             new lang_string('settings:presignedurl:presignedminfilesize', 'tool_objectfs'),
             new lang_string('settings:presignedurl:presignedminfilesize_help', 'tool_objectfs'), 0, PARAM_INT));
 
+        $settings->add(
+            new admin_setting_filetypes(
+                'tool_objectfs/signingwhitelist',
+                new lang_string('settings:presignedurl:whitelist', OBJECTFS_PLUGIN_NAME),
+                new lang_string('settings:presignedurl:whitelist_help', OBJECTFS_PLUGIN_NAME),
+                '',
+                ['onlytypes' => ['txt', 'image', 'document', 'web_file', 'video']]
```

Comment:
this might be the source of the bug where you can't save the admin setting when you tick lots of things